packages/esm-patient-attachments-app/src/attachments/attachment-thumbnail.component.tsx

```diff
@@ -1,84 +1,10 @@
-import React, { useState } from 'react';
-import styles from './attachment-thumbnail.css';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
If we now get rid of all the `@fortawesome` icons we can also remove the(se) package(s) from the *package.json*.